Description: ensure use of Debian $LDFLAGS
 Ensure use of Debian $LDFLAGS, so that hardening options get used.
Author: Moray Allan <moray@debian.org>
Last-Update: 2016-03-17
Index: GNUmakefile.in
===================================================================
--- mkcue-1.orig/GNUmakefile.in
+++ mkcue-1/GNUmakefile.in
@@ -54,7 +54,7 @@ INSTALL_SCRIPT=		${INSTALL_PROGRAM}
 all: mkcue
 
 mkcue: mkcue.o mb.o
-	${CXX} -o $@ mkcue.o mb.o
+	${CXX} -o $@ mkcue.o mb.o ${LDFLAGS}
 
 clean:
 	rm -f mkcue *.o
